test/integration/test_multisharding_two.py

```diff
@@ -157,3 +160,91 @@ def test_router_role(cluster):
         'hot': 'bbbbbbbb-bbbb-4000-b000-000000000002',
         'cold': 'cccccccc-cccc-4000-b000-000000000002'
     }
+
+
+def test_set_vshard_options_positive(cluster):
+    obj = edit_vshard_group(cluster, group = 'hot',
```

Comment:
No need to check all the options again.

The main test here is that each group can be edited separately.